src/components/routes/RoutesMap.vue

```diff
@@ -0,0 +1,385 @@
+      savedRoutes,
+      vectorLayer,
+      zoom,
+
+      tooltipCoord,
+      tooltipText,
+
+      addMapRoute,
+      centerOnCurrentLocation,
+      getRouteLengthLabel,
+      onDrawStart,
+      onDrawEnd,
+      onModifyEnd,
+      onSaveRoute,
+      onSavedRouteClick,
+      onUndo,
+      renderSavedRoute,
+      styleFunction,
+      toggleDrawEnabled,
+    };
+  },
+});
+</script>
+
+<template>
+  <div>
+    <!-- Container: Map -->
+    <div
+      class="row q-my-lg"
+      data-cy="routes-map"
+      :style="{
+        borderRadius,
+        overflow: 'hidden',
+        border: '1px solid #E0E0E0',
+      }"
+    >
+      <!-- Column: Drawn routes -->
+      <div class="col-12 col-sm-2">
+        <q-scroll-area :style="{ height: listHeight }">
+          <!-- List: Drawn routes -->
+          <q-list separator>
+            <!-- List header -->
+            <q-item class="bg-primary text-white text-weight-bold text-center">
+              <q-item-section class="text-subtitle2 text-uppercase"
+                >Vaše trasy</q-item-section
+              >
+            </q-item>
+            <!-- Item: Drawn route -->
+            <q-item
+              clickable
+              v-ripple
+              v-for="(route, index) in savedRoutes"
+              :key="`route-${index}`"
+              @click="onSavedRouteClick(route)"
+            >
+              <q-item-section v-if="route['startName'] && route['endName']">
+                <div>{{ `${route['startName']} → ${route['endName']}` }}</div>
+                <div v-if="route['length']">
+                  <small>
+                    {{ getRouteLengthLabel(route) }}
+                  </small>
+                </div>
+              </q-item-section>
+            </q-item>
+          </q-list>
+        </q-scroll-area>
+      </div>
+
+      <!-- Column: Map -->
+      <div class="relative-position col-12 col-sm-10">
+        <!-- Toolbar: Top -->
+        <routes-map-toolbar
+          :delete-enabled="deleteEnabled"
+          :draw-enabled="drawEnabled"
+          @current-position="centerOnCurrentLocation"
+          @save:route="onSaveRoute"
+          @update:delete-enabled="deleteEnabled = $event"
+          @update:draw-enabled="drawEnabled = $event"
+          @undo="onUndo"
+          data-cy="routes-map-toolbar"
+        />
+        <!-- Map -->
+        <ol-map
+          ref="mapRef"
+          :loadTilesWhileAnimating="true"
+          :loadTilesWhileInteracting="true"
+          :style="{ height: mapHeight }"
+          data-cy="routes-map-map"
+        >
+          <!-- View -->
+          <ol-view
+            ref="view"
+            :center="center"
+            :zoom="zoom"
+            :projection="projection"
+          />
+          <!-- Layer for OpenStreetMap tiles -->
+          <ol-tile-layer>
+            <ol-source-osm />
+          </ol-tile-layer>
+          <!-- Zoom controls -->
+          <ol-zoom-control zoomInLabel="➕" zoomOutLabel="➖" />
```

Comment:
According [OpenLayers doc ol/control/Zoom](https://openlayers.org/en/latest/apidoc/module-ol_control_Zoom-Zoom.html) `zoomInLabel`  and `zoomOutLabel` attribute has default value.

Reply:
Good point. Using default labels.